VmAgent.Core/MultiLogger.cs

```diff
@@ -14,11 +14,16 @@ public class MultiLogger : ILogger
     {
         private readonly ILogger _logger;
         private readonly TelemetryClient _genevaTelemetryClient;
+        private readonly ILogger _genevaOtelLogger;
 
-        public MultiLogger(ILogger logger, TelemetryClient genevaTelemetryClient)
+        public MultiLogger(ILogger logger, TelemetryClient genevaTelemetryClient, ILogger genevaOpenTelemetryLogger = null)
         {
             _logger = logger ?? throw new ArgumentException("logger cannot be null");
+
+            // We have 2 ILogger parameters (1. Logger to write logs to local file. 2.Geneva OpenTelemetry Logger)
+            // We will replace GenevaAgentchannel with OpenTelemetry, once we verify all logs both in test and prod environments.
             _genevaTelemetryClient = genevaTelemetryClient ?? throw new ArgumentException("genevaTelemetryClient cannot be null");
+            _genevaOtelLogger = genevaOpenTelemetryLogger;
```

Comment:
what are the cases that this is null?

Reply:
it will be null in prod until we enable Otel in production.